packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/multiphysics/ContextPrimalDualRelation.java

```diff
@@ -0,0 +1,16 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.multiphysics;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import java.io.Serializable;
+
+@Data
+@Accessors(chain = true)
+@TSModel
+public class ContextPrimalDualRelation implements Serializable {
+	private String primal;
+	private String dual;
+	private Object method;
```

Comment:
FWIW if these are typed as `JsonNode` it will still go to the front end as `any` but give us the option of extracting fields out of the Json in the back end without having to do strict typing.